notebooks/galaxy_redshift/redshift_fitting.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>In regards to Jwebbinar8's reference, it would be helpful to include instructions on how to use the requirements file for environment setup, as some readers may not have attended the webinar. It would also be beneficial to provide a link to the README that you are referring to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I included a reference to the jwebbinar material.

Reply:
Hi @camipacifici, can you push your changes so I can review the commits?

Reply:
Sorry, I started addressing the review and got caught up half way. I will likely finish early next week. Sorry about the wait.